docs/upgrade.html

```diff
@@ -845,7 +845,7 @@ <h5><a id="upgrade_300_notable" href="#upgrade_300_notable">Notable changes in 3
         <code>DefaultReplicationPolicy</code>, cannot prevent replication cycles based on topic names, so take care to avoid cycles when constructing your
         replication topology.
     </li>
-    <li> The original MirrorMaker (MM1) and related classes have been deprecated. Please use the Connect-based
+    <li> The original MirrorMaker (MM1) and related classes have been removed. Please use the Connect-based
```

Comment:
This doc is related to 3.0.0 release, and hence we should not change it. Could you please add new entry for 4.0.0 release?

Reply:
@chia7712 I have added the entry. Please review it to ensure its correctness. The entry has been added as follows: `<h4><a id="upgrade_4_0_0" href="#upgrade_4_0_0">Upgrading to 4.0.0 from any version 0.8.x through 3.9.x</a></h4>
    <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4.0.0</a></h5>`